core/test/utils_test.jl

```diff
@@ -261,28 +260,6 @@ end
     @test reduction_factor(-Inf, 2.0) === 0.0
 end
 
-@testitem "low_storage_factor" begin
```

Comment:
Removing a test and not adding anything back seems bad. Is the BMI coupler test not a good one to add, and make sure we that at least a part of the `step_limiter` functionality works?